impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -34,23 +35,145 @@ bool TextureGLES::IsValid() const {
 
 // |Texture|
 void TextureGLES::SetLabel(const std::string_view& label) {
-  // Unsupported.
+  if (!IsValid() || handle_.IsDead()) {
+    return;
+  }
+  reactor_->SetDebugLabel(handle_, std::string{label.data(), label.size()});
 }
 
+struct TexImage2DData {
+  GLint internal_format = 0;
+  GLenum format = GL_NONE;
+  GLenum type = GL_NONE;
+  std::shared_ptr<fml::Mapping> data;
+
+  TexImage2DData(PixelFormat pixel_format,
+                 const uint8_t* contents,
+                 size_t length) {
```

Comment:
So I wonder if g3 is OK with `std::span` yet... I recall trying to sneak it in last year but ran into problems.

Reply:
Probably not since its C++20.

Reply:
Checked and the answer is indeed no. Still just `absl::Span` everywhere. FML has a dependency on abseil, but not sure if we want to make a habit of using compat shims like that.

Reply:
I also suppose the only cpp20 features that really get turned on are syntax changes (like designated initializers) and not stdlib, since abseil takes care of the stdlib coverage...